cmd/chart.go

```diff
@@ -117,7 +117,13 @@ func moveChart(cmd *cobra.Command, args []string) error {
 			Rules: *targetRewriteRules,
 		},
 	}
-	moveRequest.Source.Chart.Local = &mover.LocalChart{Path: args[0]}
+	inputChartPath := args[0]
+	if mover.IsIntermediateBundle(inputChartPath) {
+		cmd.Printf("Detected intermediate bundle as input at %s\n", inputChartPath)
```

Comment:
nitpick. This message might be unnecessary and inconsistent since we do not add a message when it's local

In any case, I'd simplify it `cmd.Println("Intermediate bundle provided")`, the path to the argument might not be that useful

Reply:
will fix

I think I should add a message when it is chart as well.
What do not think is ok is for us to do auto detection and not show the user what was detected. For instance, if you were expecting a bundle to work and it fails it falls back to chart and complains it is not a tgz... which is puzzling.

Reply:
fixed